.github/workflows/main.yml

```diff
@@ -19,22 +19,13 @@ jobs:
             major: 3.15
           - flavor: centos
             major: 7
-          - flavor: debian
-            major: 11
-          - flavor: ubuntu
-            major: 20.04
```

Comment:
why are we going support for these? (or why we added support beforehand?)

Reply:
added for evaluation in order to replace centos7 but the security scanning results are surprisingly poor.

Reply:
create another issue to remove them thanks